language/de-DE/tpl_cassiopeia.sys.ini

```diff
@@ -7,6 +7,7 @@
 CASSIOPEIA="Cassiopeia Frontend Template"
 TPL_CASSIOPEIA_MOD_MENU_LAYOUT_DROPDOWN-METISMENU="Dropdown"
 TPL_CASSIOPEIA_POSITION_BANNER="Banner"
+TPL_CASSIOPEIA_POSITION_BELOW-TOP="Below Top"
```

Comment:
Translation missing

Reply:
Richtig, ich hatte in meinem Kommentar schon geschrieben:

> Hier fällt mir keine wirklich gute Übersetzung ein:
TPL_CASSIOPEIA_POSITION_BELOW-TOP="Below Top"

> Kann das so bleiben?

Reply:
Nee, dass müssen wir übersetzen.

Reply:
Wird diese Übersetzung noch an anderer Stelle benutzt als beim Auswahlmenü für die Modulposition?
![Position_A](https://user-images.githubusercontent.com/853497/120118338-5592c480-c192-11eb-820d-4e4c43edf70f.png)

Wenn ja, dann kann ich keinen Nutzen erkennen – im Gegenteil, je nach Seitenbreite des Fensters und der Länge der Übersetzung bricht die Zeile  unkontrolliert um und erschwert die Lesbarkeit. 
![Position_B](https://user-images.githubusercontent.com/853497/120118343-5b88a580-c192-11eb-98a4-9801a2d90578.png)

Ein weiterer Nachteil die Modulpositionen zu übersetzen ist in den nächsten Bildern zu erkennen. Weder in der Vorschau noch in der Modul-Liste wird der übersetzte Begriff verwendet. Zwei Begriffe für die gleiche Position schafft nur Verwirrung.

![Vorschau](https://user-images.githubusercontent.com/853497/120118996-9dffb180-c195-11eb-96d1-d383867255b7.png)
![Modul-Liste](https://user-images.githubusercontent.com/853497/120119001-a6f08300-c195-11eb-9565-5fec3191868c.png)

Reply:
Schaue ich mir an aber guter Einwand. Wird geprüft.

Reply:
Nachdem man in der Suche nach einer Modulposition auch nach dem "Übersetzten" suchen kann, sollten wir es auf deutsch lassen. Das kennen die Nutzer auch bereits von den bisherigen Core-Templates.